run

```diff
@@ -8,6 +8,11 @@ if [[ -d "/var/task/snowalert/venv/bin" && -d "/var/task/snowalert/src" ]]; then
     cd /var/task/snowalert/src/webui/backend
     python webui/app.py
 
+  elif [ "$1" == "process-baseline" ];  then
```

Comment:
Its not a baseline, and we don't want it running with the rest of the baselines. It runs once a week rather than once a day and handles two data sources and a very large chunk of data.